code/game/machinery/camera/tracking.dm

```diff
@@ -146,10 +146,6 @@
 		for (var/j = 1 to i - 1)

```

Comment:
This /proc/camera_sort should probably be rework to be sorted instead and storing lists like this which need to be ordered as a GLOB for efficency and processing storage but that is what I will do for the next PR. (I have further plans which will probably include doing this)